docker/kafka-setup/kafka-setup.sh

```diff
@@ -6,6 +6,8 @@
 
 : ${DATAHUB_ANALYTICS_ENABLED:=true}
 
+: ${KAFKA_HEAP_OPTS:=-Xmx64M}
```

Comment:
We shouldn’t hard code define the max heap size of the jvm. The docker image should instead use jre11 which by default consumes 75% of the available memory for the container.

Reply:
@pedro93 - This is for running cli commands. It should not need 75% of the memory ever, settings fixed amount of memory per *cli command* is reasonable.

Reply:
Default max is 25%, not 75%: https://docs.oracle.com/en/java/javase/11/gctuning/ergonomics.html#GUID-DA88B6A6-AF89-4423-95A6-BBCBD9FAE781